corehq/toggles/__init__.py

```diff
@@ -742,6 +742,18 @@ def _ensure_valid_randomness(randomness):
     [NAMESPACE_DOMAIN],
 )
 
+FORM_LINK_ADVANCED_MODE = StaticToggle(
```

Comment:
will this be a temporary FF? We really are pretty quick to add new ones aren't we?

Reply:
I don't see an alternative, unfortunately (though I'm open to ideas!).  This feature is generally available for some reason, even though it's pretty brittle.  I was thinking about trying to integrate the advanced mode into the normal manual mode as mentioned in the description, and may do that if delivery advocates for it.  However, I don't think it's a good idea to increase the set of linkable targets to GA audiences, knowing that they don't work unless you are able to figure out the "advanced mode" linking stuff.

Reply:
That makes sense. Curious if you would see any value in have a parent FF like "experimental" to group these kinds of FF and give them a bit more context?

Reply:
You mean like a separate feature flag tag?  I don't know, would something in the description make sense?  This seems fairly in line with other custom feature flags to me - what's the information you're thinking about conveying?